docs/api/commands/origin.mdx

```diff
@@ -406,54 +406,32 @@ into the callback.
 
 ### Dependencies / Sharing Code
 
-[ES module dynamic `import()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#dynamic_imports)
-and/or
+[`Cypress.require()`](/api/cypress-api/require) can be utilized to include
+[npm](https://www.npmjs.com/) packages and other files. It is functionally the
+same as using
 [CommonJS `require()`](https://nodejs.org/en/knowledge/getting-started/what-is-require/)
-can be used within the callback to include [npm](https://www.npmjs.com/)
-packages and other files.
+in browser-targeted code.
```

Comment:
we can't use outside of origin right?
```suggestion
in browser-targeted code that will execute in a cross-origin context.
```

Reply:
This sentence is more about how its usage compares to CommonJS `require` and there's no real difference when it comes to cross-origin context. I'm mostly trying to give a point of reference and briefly communicate that if you can do `require('lodash')`, you can also do `Cypress.require('lodash')`, without getting too much into the nitty gritty of it. The `Cypress.require` doc itself gets more into the restrictions and specifics of the API and where you can use it.

I updated the previous sentence to make it clear this only applies within the `cy.origin()` callback.